frontend/src/components/pages/RegistrantExperience/RegistrationSteps/index.tsx

```diff
@@ -289,6 +326,8 @@ const RegistrationSteps = ({
     } else if (desiredStep < 0) {
       onClickBack();
     } else {
+      // Append the optional clauses to all the campers before registering the campers
+      appendOptionalClausesToCampers(waiverInterface.optionalClauses);
```

Comment:
Is this different from `mapToCreateCamperDTO`? - https://github.com/uwblueprint/focus-on-nature/blob/dev/frontend/src/utils/CampUtils.ts#L328

https://github.com/uwblueprint/focus-on-nature/blob/f4268bfa7d8e447eaa758797bc1428af20431048/frontend/src/utils/CampUtils.ts#L368-L374